mobie/mobie.pq

```diff
@@ -54,34 +85,73 @@ mobieType = type function (
 		Documentation.FieldCaption = "Server Name",
 		Documentation.FieldDescription = { "DataBox database server name for connection string" }
 	]),
-	Database as (type text meta [
+	optional Database as (type text meta [
 		Documentation.FieldCaption = "Database Name",
-		Documentation.SampleValues = { "DataBox" }
+		Documentation.SampleValues = { "DataBox. Required for SQL Server. Unused for Oracle." }
 	]),
-	DataBoxSchemaName as (type text meta [
+	optional DataBoxSchemaName as (type text meta [
 		Documentation.FieldCaption = "Schema Name",
-		Documentation.SampleValues = { "dbo" }
+		Documentation.SampleValues = { "Defaults to dbo for SQL Server. Must be specified for Oracle." }
+	]),
+	optional FDASPROPName as (type text meta [
+		Documentation.FieldCaption = "FDASPROP Table Name",
+		Documentation.SampleValues = { "Blank for default FDASPROP" }
 	]))
 	as table meta [
 		Documentation.Name = "mobie® DataBox",
 		Documentation.LongDescription = "Returns a table of all DataBox tables stored in the specified DataBox database"
 	];
 		
+GetMobieIdentity = (Provider as text, ConnectionString as text, DatabaseType as text, dbo_FDASPROP as table) =>
+	let
+		SessionUser =
+			if DatabaseType = "SQL Server" then
+				AdoDotNet.Query(Provider, ConnectionString, "SELECT UPPER(TRIM(SYSTEM_USER)) AS UserName"){0}[UserName]
```

Comment:
I'm getting an error possibly associated with this call: 
![image](https://user-images.githubusercontent.com/2396671/134559263-35ae2393-439d-48e7-899c-5595ec030797.png)

Reply:
My database version is 13.0.5103.6

Reply:
Fixed.